nncf/torch/tensor_statistics/collectors.py

```diff
@@ -173,17 +173,10 @@ def quantile(
     ) -> List[NNCFTensor]:
         device = tensor.device
         # See https://github.com/pytorch/pytorch/issues/61582
-        if not isinstance(axis, int):
-            result = torch.tensor(
-                np.quantile(tensor.tensor.detach().cpu().numpy(), q=quantile, axis=axis, keepdims=keepdims)
-            )
-        else:
-            result = torch.quantile(
-                tensor.tensor,
-                torch.tensor(quantile, dtype=tensor.tensor.dtype, device=tensor.tensor.device),
-                axis,
-                keepdims,
-            )
+        # https://github.com/pytorch/pytorch/issues/64947
+        result = torch.tensor(
+            np.quantile(tensor.tensor.detach().cpu().numpy(), q=quantile, axis=axis, keepdims=keepdims)
+        )
```

Comment:
If the shortcomings of the torch implementation are limited to the CPU version, then the code should only fall back to the numpy implementation if the original tensor device is CPU.

Reply:
I'm now using `torch.quantize` in case it is possible